benchmark/sirun/get-results.js

```diff
@@ -131,7 +131,7 @@ async function getResults (gitCommit) {
   return buildData
 }
 
-async function main () {
+function main () {
```

Comment:
I assume this was never caught because we never run this file directly, so `main()` is never called 😆.

The actual issue here is that `getResults()` should have an `await` in front of it.

Reply:
Fixed 👍